source/portable/NetworkInterface/DriverSAM/NetworkInterface.c

```diff
@@ -204,14 +204,17 @@ static void hand_tx_errors( void );
 static uint16_t prvGenerateCRC16( const uint8_t * pucAddress );
 static void prvAddMulticastMACAddress( const uint8_t * ucMacAddress );
 
+/* Checks IP queue, buffers, and semaphore and logs diagnostic info if configured */
+void vCheckBuffersAndQueue( void );
```

Comment:
I believe its better to keep `vCheckBuffersAndQueue` as a static function as its only used here.

Reply:
Agreed. Changed to a static funciton in [a715759](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/1016/commits/a715759cfd6c6908f9ac183a04c6e954c9a72c99)